libraries/RainMaker/src/RMakerParam.h

```diff
@@ -47,5 +47,6 @@ class Param
         esp_err_t addUIType(const char *ui_type);
         esp_err_t addBounds(param_val_t min, param_val_t max, param_val_t step);
         esp_err_t updateAndReport(param_val_t val);
+        esp_err_t addValidStrList(const char** str_list);
```

Comment:
your function declaration does not match the implementation. In the implementation you have two arguments: `const char **string_list, uint8_t count`. Probably you missed the second one.